src/stars.coffee

```diff
@@ -60,8 +61,22 @@ class Stars extends Command
   installPackages: (packages, callback) ->
     return callback() if packages.length is 0
 
-    commandArgs = packages.map ({name}) -> name
-    new Install().run({commandArgs, callback})
+    toName = ({name}) -> name
+
+
+    list = new List()
+    listOptions =
+      argv:
+        dev: true
+        bare: true
+
+    list.listUserPackages listOptions, (error, userPackages) ->
+      userPackages = userPackages.map(toName)
+
+      commandArgs = packages.map(toName).filter (name) ->
+        return name not in userPackages
+
+      new Install().run({commandArgs, callback})
```

Comment:
@AsaAyers That's a good question. I _think_ you could use [this approach](https://github.com/AsaAyers/apm/blob/e144f99e01be02322d07f0799bb34b6384c7882f/spec/develop-spec.coffee#L25) to spy on `Install.run()` (but still allow the call to go through). What do you think?